universal-login-relayer/lib/http/routes/wallet.ts

```diff
@@ -25,13 +25,14 @@ const deploymentHandling = (deploymentHandler: DeploymentHandler) =>
   async (data: {body: DeployArgs & {applicationInfo: ApplicationInfo}}, req: Request) => {
     const {applicationInfo, ...deployArgs} = data.body;
     const deviceInfo = getDeviceInfo(req, applicationInfo);
-    const transaction = await deploymentHandler.handleDeployment(deployArgs, deviceInfo);
-    return responseOf(transaction, 201);
+    const hash = await deploymentHandler.handleDeployment(deployArgs, deviceInfo);
+    return responseOf(hash, 201);
```

Comment:
Breaking change in relayer's API - used to return ether's transaction object, now returns hash that identifies the queued deployment

Reply:
Should we return status here instead?